incubator/etcd/README.md

```diff
@@ -38,15 +38,16 @@ The following tables lists the configurable parameters of the etcd chart and the
 | Parameter               | Description                        | Default                                                    |
 | ----------------------- | ---------------------------------- | ---------------------------------------------------------- |
 | `Name`                  | Spark master name                  | `etcd`                                                     |
-| `Image`                 | Container image name               | `k8s.gcr.io/etcd-amd64`                      |
-| `ImageTag`              | Container image tag                | `2.2.5`                                                    |
+| `Image`                 | Container image name               | `nearform/etcd`                      |
```

Comment:
I understand your point on using a non-root user for openshift. Since the image in use today is managed by k8s itself I'd like to chase that down first.

Reply:
I was unable to track the person down. Will dig into this as is now.

Reply:
Is there anything needed from my side? I'll have some time next week.